### ISSUE TITLE
[5.69] btmgnt not accepting -i hci0

### ISSUE THREAD

**Issue by htot** [bluez/bluez#694]
With BCM43341B0 (Intel-Edison) it doesn't do anything until public-addr is loaded. So even bd_addr gives an error:
```
root@yuna:~# rfkill unblock bluetooth
root@yuna:~# bdaddr -i hci0 -r 98:4F:EE:02:D4:D9 
Invalid device: Network is down
```
Up to now (5.69) the following always worked:
```
btmgmt -i hci0 public-addr  98:4F:EE:02:D4:D9
```
but now this returns:
```
root@yuna:~# btmgmt -i hci0 public-addr  98:4F:EE:02:D4:D9
Unable to open hci0: No such file or directory (2)
hci0 Set Public Address complete, options: 
```
in fact it doesn't seem to accept any argument for -i.
What does work is:
```
root@yuna:~# btmgmt
[mgmt]# select hci0
Selected index 0
[hci0]# public-addr 98:4F:EE:02:D4:D9
hci0 Set Public Address complete, options: 
hci0 removed (unconfigured)
hci0 new_settings: br/edr 
hci0 added
hci0 new_settings: ssp br/edr 
hci0 new_settings: ssp br/edr le 
hci0 new_settings: ssp br/edr le secure-conn 
hci0 name changed: yuna
hci0 device_flags_changed: E8:78:29:C4:EA:37 (BR/EDR)
     supp: 0x00000001  curr: 0x00000000
[hci0]# quit
```
Now:
```
root@yuna:~# rfkill unblock bluetooth
root@yuna:~# bdaddr -i hci0
Manufacturer:   Broadcom Corporation (15)
Device address: 98:4F:EE:02:D4:D9 (Intel Corporate)
```

Also not working:
```
btmgmt --init-script=test-script.bt
```
and
```
btmgmt << EOF
select hci0
public-addr  98:4F:EE:02:D4:D9
EOF
```

**Comment by htot** [comment]
The problem `btmgmt -i hci0` throwing error `Unable to open hci0: No such file or directory (2)` is reproducible on Ubuntu 23.10 (Mantic Minotaur) with `btmgmt` v5.68.

**Comment by htot** [comment]
Sorry this is a duplicate of https://github.com/bluez/bluez/issues/639
and https://bugs.launchpad.net/ubuntu/+source/bluez/+bug/2041496

and fixed in 303925b28110469ad002ac19ce0eb9c84d6aceb2
